File: platforms/kotlin/examples/scene/light/set_color.kt
import org.fragmentcolor.*

val lamp = Light.point(listOf(0.0f, 2.0f, 0.0f), listOf(1.0f, 1.0f, 1.0f))

// Warm-tint the lamp later — every Pass that absorbed """lamp""" sees the
// color on the next render.
lamp.setColor(listOf(1.0f, 0.7f, 0.4f))

File: platforms/python/examples/scene/light/set_color.py
from fragmentcolor import Light

lamp = Light.point([0.0, 2.0, 0.0], [1.0, 1.0, 1.0])

# Warm-tint the lamp later — every Pass that absorbed `lamp` sees the
# new color on the next render.
lamp.set_color([1.0, 0.7, 0.4])